### ISSUE TITLE
Images are not displayed in Preview Mode

### ISSUE THREAD

**Issue by v-yvric** [mjbvz/vscode-github-markdown-preview-style#22]
In a previous release, the Preview Mode would display the image instead of the link.  Was a preference erased with one of the recent upgrades? 

![image](https://user-images.githubusercontent.com/32719282/39592006-61b1e546-4eba-11e8-9b5e-ac2e794f2bbf.png)

**Comment by mjbvz** [comment]
This is unrelated to this extension. VS Code will only load images from inside your  current workspace. If you opened a loose md file, you cannot use `../` to access images

**Comment by v-yvric** [comment]
I created a workspace, and added the folders for the docs that I was working on  and the media, and the image still will not load.
I didn’t have a workspace previously; is it possible that it is not created correctly?

**Comment by v-yvric** [comment]
I have installed a few more editions of VS, and the images are still not displaying.    I found 
![markdown tags 2](https://user-images.githubusercontent.com/32719282/41802239-40ff07ec-7634-11e8-82cd-cd384c0f9036.png) and experimented with one or two properties, but none of them seem to turn the images back on.  Ctrl-clicking on the image to display it in a new window works, but is not the answer I am seeking.
